documentation/collection/integration/oauth/reference/provider-api.md

```diff
@@ -99,6 +99,7 @@ type ProviderSession = {
 | Patreon  | [`PatreonUser`](/oauth/providers/patreon#patreonuser)    |
 | Reddit   | [`RedditUser`](/oauth/providers/reddit#reddituser)       |
 | Twitch   | [`TwitchUser`](/oauth/providers/twitch#twitchuser)       |
+| Twitch   | [`LinkedInUser`](/oauth/providers/linkedin#linkedinuser) |
```

Comment:
```suggestion
| LinkedIn   | [`LinkedInUser`](/oauth/providers/linkedin#linkedinuser) |
```

Reply:
Silly me. This happens when you do copy&paste :)